src/main/java/com/github/fge/jsonschema/core/load/configuration/LoadingConfiguration.java

```diff
@@ -89,19 +89,11 @@
     final URITranslatorConfiguration translatorCfg;
     
     /**
-     * Should we cache schemas
+     * Cache size
      *
      * <p>Note that this do not affect preloaded schemas; these are always
      * cached.</p>
      */
-    final boolean enableCache;
```

Comment:
My fear with removing this would be if there are other folks that are using this configuration setting we will break them.  If you want to can we add an  @deprecated to the method that sets it and translate a false value to maximumSize 0.